launcher/internal/signaturediscovery/client.go

```diff
@@ -80,13 +80,17 @@ func (c *Client) FetchImageSignatures(ctx context.Context, targetRepository stri
 func (c *Client) pullTargetImage(ctx context.Context, targetRepository string) (containerd.Image, error) {
 	targetImageRef := fmt.Sprint(targetRepository, ":", formatSigTag(c.OriginalImageDesc))
 
+	// Pull signature image from a public repository.
+	if c.remoteResolverFetcher == nil {
+		return c.imageFetcher(ctx, targetImageRef)
```

Comment:
same here, would you be open to calling this pullImage?
Or fetch image? Something to denote this is a func that is getting the image.

Reply:
This method aims to fetch signature image based on the digest of the signed container, so `pullSignatureImage` seems more appropriate here.